src/vita-make-fself.c

```diff
@@ -156,6 +162,14 @@ int main(int argc, const char **argv) {
 
 	uint32_t offset_to_real_elf = HEADER_LEN;
 
+	if (self_type == 0xB) { // Security Module
```

Comment:
how about making the magic number or enum for it?

Reply:
Done.